### ISSUE TITLE
NTR: Prenatal ultrasound: diaphragm flattened (Lab: ultrasound & echocardiogram)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8686]
New term request Prenatal ultrasound: diaphragm flattened (elthat_101002181953): Downward displacement; eversion of diaphragm included  UMLS: 0743169

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000340
- label: Prenatal flattened diaphragm
- parents: [Abnormal fetal pulmonary morphology (HP:0034243)](https://hpo.jax.org/app/browse/term/HP:0034243)
- pmid: PMID:30319217
- def: The diaphragm normally projects up into the thoracic cavity. In the presence of malformations such as enlarged lungs, the diaphragm may appear abnormally flat or even everted (with the middle of the diaphragm projecting downwards into the abdominal cavity).
- comment: Serr Figures 1 and 2 of PMID:30319217.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8686
- synonyms:
